frontend/src/app/_models/globals.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from "@angular/core";
+import notify from 'devextreme/ui/notify';
+import { DatePipe } from "@angular/common";
+
+
+@Injectable()
+export class Globals {
+
+  public toastTimeDuration: number = 1000;
+  public toastSuccessType: string = "success";
```

Comment:
personal preference. Try to avoid using the technology in the name for something like this.
If you changed from toast to another system you would still want your "SuccessType" to be "success".
Consider naming it notification{property detail here}